src/components/qas/questions/qaFillableTable/FillableTableRangeInput.jsx

```diff
@@ -187,6 +188,7 @@ FillableTableRangeInput.propTypes = {
   focusCallback: PropTypes.func,
   answerHandler: PropTypes.func,
   answer: PropTypes.object,
+  t: PropTypes.func,
 };
 
-export default FillableTableRangeInput;
+export default withTranslation('interface')(FillableTableRangeInput);
```

Comment:
`withTranslation` can also be set to use a default namespace, in this case it is `interface`